frontend/src/views/web/challenge-list.html

```diff
@@ -1,6 +1,6 @@
 <section class="ev-sm-container ev-view challenge-container">
     <!-- ongoing challenges -->
-    <div class="challenge-page-title" id = "ongoing-challenges"><strong class="text-med-black fs-18">Ongoing Challenges</strong></div>
+    <div class="challenge-page-title" id = "ongoing-challenges"><strong class="text-med-black fs-18">Ongoing Challenges hdfhdhj fhdsjfhjdhkj</strong></div>
```

Comment:
There is some typo here. Please remove the unnecessary characters here: `hdfhdhj fhdsjfhjdhkj`